.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```

Comment:
`/bin/sh` is the preferred shebang I believe

Reply:
I wouldn't want to be on that jury...! However, husky creates the hooks like this (i.e. with `husky add pre-commit "yarn --silent lint-staged -r"`).